### DISCUSSION TITLE
Grott and OpenHab with "Growatt Solar Inverter Binding"

### DISCUSSION THREAD

**Discussion by bijama1** [johanmeijer/grott#549] [General]
I am using the **Growatt Solar Inverter Binding** (https://community.openhab.org/t/growatt-solar-inverter-binding-4-0-0-4-2-0/150600)

This requires, according to [Readme.md](https://github.com/andrewfg/openhab-addons/releases/download/v4.1/README.md) to use the following setting:

[extension] // enable the 'grottext' extension
extension = True
extname = grottext
extvar = {"url": "http://127.0.0.1:8080/growatt"} // or ip address of openHAB (if remote) 

This does NOT work for me.

I had to use:

extvar = {"ip": "127.0.0.1", "port" : "8080"}

AND changed in grottext.py:

url = "http://" + conf.extvar["ip"] + ":" + str(conf.extvar["port"] + **"/growatt"**)

**Comment by johanmeijer** [comment]
I changed the grottext.py (see #356). does this helps?
